dragonfly_doe2/doe/floor_space.py

```diff
@@ -0,0 +1,149 @@
+from dragonfly.room2d import Room2D
+from dragonfly.story import Story
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class Wall:
+    """*.inp Wall object.
+    Example:
+        .. code-block::f#
+            "simple_example_dfb_Floor1_Room1 Wall_1" = EXTERIOR-WALL
+               CONSTRUCTION    = "EWall Construction"
+               LOCATION        = SPACE-V1
+               ..
+    """
+    name: str = None
+    location_index: int = None
+    wall_con_name: str = None
```

Comment:
It's probably me but I really suggest keeping the names simple and adding a good docstring to say what is what. Class arguments naming is different from naming the variables inside your code.

```suggestion
    location: int
    construction: str
```